packages/DraftEditor/src/DraftPlugins/InlineToolbar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Button from "@blaze-react/button";
+import React, { useEffect } from "react";
+
+const Pickers = (components: any[]) => {
+  return (props: any) => {
+    useEffect(() => {
+      window.addEventListener("click", onWindowClick);
+      return () => window.removeEventListener("click", onWindowClick);
+    }, []);
+
+    const onWindowClick = () => props.onOverrideContent(undefined);
+
+    return (
+      <div onClick={onWindowClick}>
+        {components.map((DraftButton, i) => (
+          <DraftButton key={i} {...props} />
```

Comment:
It would be better not use the index as a key. If its for some specific reason, ignore the comment

Reply:
Fixed, thanks